### ISSUE TITLE
"include" deprecated in tasks

### ISSUE THREAD

**Issue by toozej** [toozej/ansible#40]
[DEPRECATION WARNING]: The use of 'include' for tasks has been deprecated. Use 
'import_tasks' for static inclusions or 'include_tasks' for dynamic inclusions.
 This feature will be removed in a future release. Deprecation warnings can be 
disabled by setting deprecation_warnings=False in ansible.cfg.
[DEPRECATION WARNING]: include is kept for backwards compatibility but usage is
 discouraged. The module documentation details page may explain more about this
 rationale.. This feature will be removed in a future release. Deprecation 
warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.

**Comment by toozej** [comment]
Should be able to use something like:
```ansible
- hosts: webservers
  tasks:
  - debug:
      msg: "before we run our role"
  - import_role:
      name: example
  - include_role:
      name: example
  - debug:
      msg: "after we ran our role"
```
from https://docs.ansible.com/ansible/latest/playbooks_reuse_roles.html. More info https://docs.ansible.com/ansible/latest/playbooks_reuse.html#dynamic-vs-static

**Comment by toozej** [comment]
Going to backburner this until close to when Ansible 2.8 comes out, since the way I'm including roles in `tests/test.yml` works for now and I'm not totally sure how to use the `include_role` section above. (Tried it and it wasn' just copy/pasta sadly)

**Comment by toozej** [comment]
Seems to be solved by Ansible 2.5 https://travis-ci.org/toozej/ansible/jobs/360011287
